README.md

```diff
@@ -1,6 +1,8 @@
 # Falco Rules
 
-[![Latest release](https://img.shields.io/github/v/release/falcosecurity/rules?style=for-the-badge)](https://github.com/falcosecurity/rules/releases/latest) [![Docs](https://img.shields.io/badge/docs-latest-green.svg?style=for-the-badge)](https://falco.org/docs/rules)  [![Rules Overview](https://img.shields.io/badge/docs-latest-green.svg?label=Rules%20Overview&style=for-the-badge)](https://falcosecurity.github.io/rules/) [![Style Guide](https://img.shields.io/badge/docs-latest-green.svg?label=Style%20Guide&style=for-the-badge)](https://falco.org/docs/rules/style-guide/) 
+[![Latest release](https://img.shields.io/github/v/release/falcosecurity/rules?label=Latest%20Rules%20Release&style=for-the-badge)](https://github.com/falcosecurity/rules/releases/latest) [![Compatible Falco release](https://img.shields.io/github/v/release/falcosecurity/falco?label=Compatible%20Falco%20Release&style=for-the-badge)](https://github.com/falcosecurity/falco/releases/latest) 
```

Comment:
It would be nice to use https://github.com/falcosecurity/rules/blob/main/.github/FALCO_VERSIONS as input to generate the list of compatible versions.

Not a blocker for now. Just an idea.

Reply:
Yes this would be very cool! Would suggest regarding this PR finished and perhaps @leogr and @LucaGuerra you could open a small follow up PR once we have the next Falco version released. That PR could then also include more links and cross-references to new docs we have yet to create.